include/variables.h

```diff
@@ -1188,14 +1188,9 @@ extern UNK_PTR D_801D5FE0;
 // extern UNK_TYPE1 D_801D5FE4;
 extern s32 gAudioCtxInitalized;
 // extern UNK_TYPE4 D_801D5FEC;
-// extern UNK_TYPE4 D_801D5FF0;
-// extern UNK_TYPE4 D_801D5FF4;
-// extern UNK_TYPE1 D_801D5FF8;
-// extern UNK_TYPE4 D_801D5FFC;
-extern UNK_PTR D_801D6000;
 // extern UNK_TYPE4 D_801D6190;
 // extern UNK_TYPE4 D_801D6194;
-// extern UNK_TYPE1 D_801D6200;
+extern u8 D_801D6200[1024];
```

Comment:
```suggestion
extern u8 D_801D6200[0x400];
```
Since it's not really an array from the looks of it. Or maybe as u64 since that's the only way it's used?

Reply:
This would be good to double check with @Thar0 since they've looked at this data specifically. `u8` vs. `u64`? (Sorry for yet another ping)